msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Overview of publish your app
+description: This is an overview of how to package and publish your app.
+ms.topic: conceptual
+ms.author: lajanuar
+keywords: teams apps Upload
+---
+
+# Publish your apps
+Apps that you develop must be made available to an individual, team, or an organization for the users to use them. You achieve this by publishing your apps. There is a complete process to publish your apps that includes, selecting methods of distribution, packaging and uploading, and finally publishing.
+
+This document provides an overview of the steps you must take to publish your app successfully to an App Store or AppSource, formerly Office Store.
+
+## Distribute your apps
+You can choose to distribute your apps to individuals, teams, or organizations. When distributing the apps to organizations, you must ensure that you adhere to the compliance policies of the selected organization. There are different methods to distribute your app. For more information, see [distribute your app](overview.md).
+
+## Create your app package
+After selecting the method to distribute your app, you need to create package that includes your app's configurations and core assets. For more information, see [create your app package](../build-and-test/apps-package.md).
+
+## Upload your app for a team or user
+You can upload your app package to a team or a conversation through the App Store. You can also upload it using the AppSource. For more information, see [upload your app package](apps-upload.md).
+
+## Publish your app to your organization
+The final step is to publish your app to your organization, such that the users can start using your app. For more information, see [publish your app](https://docs.microsoft.com/MicrosoftTeams/manage-apps?toc=%2Fmicrosoftteams%2Fplatform%2Ftoc.json&bc=%2FMicrosoftTeams%2Fbreadcrumb%2Ftoc.json).
```

Comment:
@KirtiPereira This isn't a "final step" but one of the three equal publishing options.